alby/alby_oauth_service.go

```diff
@@ -262,7 +261,7 @@ func (svc *albyOAuthService) DrainSharedWallet(ctx context.Context, lnClient lnc
 
 	amountSat := int64(math.Floor(
 		balanceSat- // Alby shared node balance in sats
-			(balanceSat*(8/1000))- // Alby service fee (0.8%)
+			(balanceSat*(8.0/1000.0))- // Alby service fee (0.8%)
```

Comment:
@rolznz I think that could be another reason why the migration doesn't work currently. Without the `.` the calculation result is an integer, which will always be 0 (so the full balance will be deducted.

Reply:
@reneaaron nice catch!